docs/UserGuide.md

```diff
@@ -359,40 +395,140 @@ A person can have any number of tags and insurance policies (including 0).
+
+[Return to Table of Contents](#table-of-contents)
+<br><br>
 
-**Format**: `list [-n] [-p] [-e] [-a] [-i] [-t] [-m]`
+### <span style="color:#f75c66">Contact Viewing Features</span>
+Contact viewing features allow you to view your client contacts in different ways. For example,
+you may reorder your contact list or look for specific clients.
+
+### <span style="color:#f75c66"><code>list</code>: List all clients</span>
+
+**Purpose**: Shows a list of all clients in ClientBook. You may use optional identifiers in conjunction with the minus (-)
+symbol to select which client details you want to see, so that you can have a clutter free view of your client contacts.
+
+**Format**: `list [-p] [-e] [-a] [-i] [-m]`
 
 **Examples**: 
 *  `list` without any specified identifiers shows a list of all clients and all their information.
-  * `list`
-    
-    ![list](images/list.png)
+
+![list](images/list.png)
     
-*  One or more identifiers can be used to make `list` only show the specified information. The following command shows a list of all clients and their phone number and insurance policy number.
-   * `list -p -i`
+*  One or more identifiers can be used to make `list` only show the specified information. The following command 
+   shows a list of all clients and their phone number and insurance policy number. A client's name and tags will
+   be shown regardless of the identifiers that you specify.
+   
+`list -p -i`
     
-     ![list phone](images/list-phone-policy-annotate.png)
+![list phone](images/list-phone-policy-annotate.png)
 
 [Return to Table of Contents](#table-of-contents)
 <br><br>
 
   
-### `find`: Search for client contact based on keywords
+### <span style="color:#f75c66"><code>find</code>: Search for client contact based on keywords</span>
 
-**Purpose**: Finds and displays all clients whose field (name, phone, email, address, tags, insurance policy) contains any of the given keywords.
-Optional identifiers can be added to show the list of matched clients with only the specified field(s), similar to the [`list`](#list-list-all-clients) command.
+**Purpose**: Finds client contacts based on the attribute and keywords that you specify.
+You may use optional identifiers in conjunction with the minus(-) symbol to limit the details shown, similar to the [`list`](#list-list-all-clients) command.
 
 **Format**: `find IDENTIFIER/KEYWORD [& KEYWORDS]…​ [-IDENTIFIER]…​`
 
+* The `IDENTIFIER` specifies which attribute of the client you want to search for. For example, if you want to search for a client contact
+  by name, use the `n` identifier.
+    * For more information about the identifiers for each field, refer to [What information can we store for each client contact?](#what-information-can-we-store-for-each-client-contact).
+* You may not use multiple attributes as your search criteria e.g. you cannot search for a client using address and name at the same time.
+  A command like `find n/David a/Botanic gardens` is invalid.
```

Comment:
Maybe we can bold "may not use multiple attributes"? So that it looks better. Just my opinion

Reply:
I feel that it may be too much bolding around the area.